### PR TITLE
init-system-helpers: Add init-system-helpers package.

### PR DESCRIPTION
**Pull request by AkitoTakada**
I confirmed that the following commands work.

```
# service bootlogd status
/sbin/bootlogd (pid 1492) is running...
```

### CODE REVIEW

File: recipes-debian/init-system-helpers/init-system-helpers_debian.bb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# base recipe: meta-debian/recipes-debian/init-system-helpers/init-system-helpers_debian.bb
```
Comment (masami256) [inline recipes-debian/init-system-helpers/init-system-helpers_debian.bb:1]: Is this correct path? I couldn't init-system-helpers directory in meta-debian.

Comment (AkitoTakada) [reply recipes-debian/init-system-helpers/init-system-helpers_debian.bb:1]: Files deleted from meta-debian. Created based on the recipe before deletion.

File: recipes-debian/init-system-helpers/init-system-helpers_debian.bb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+LICENSE = "GPLv3+ & BSD"
+LIC_FILES_CHKSUM = "file://script/deb-systemd-invoke;beginline=3;endline=30;md5=42d36293e53d929566aef24e3e3b9ef8"
+
+do_install() {
+	install -d ${D}${bindir}
+	install -d ${D}${datadir}/dh-systemd
+	install -d ${D}${datadir}/perl5/Debian/Debhelper/Sequence
+	install -d ${D}${datadir}/debhelper
+        install -d ${D}${datadir}/man/man8
```
Comment (masami256) [inline recipes-debian/init-system-helpers/init-system-helpers_debian.bb]: @AkitoTakada  I'll ask small changes. please align indent please?

Comment (AkitoTakada) [reply recipes-debian/init-system-helpers/init-system-helpers_debian.bb]: We have indented as pointed out.

Comment (masami256) [reply recipes-debian/init-system-helpers/init-system-helpers_debian.bb]: thanks!

**Pull request merged**